web-dapp/src/components/IndexPage.js

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 
+const REACT_APP_PRICE_ETH = process.env.REACT_APP_PRICE_ETH;
```

Comment:
The `_ETH` suffix is misleading if the price can be paid in POA.

Reply:
I was thinking to make it clear that `PRICE_ETH` is in "main units" like eth/keth/poa/... as opposed to `priceWei` which is in wei. Maybe some better name comes to mind?

Reply:
I see. Maybe `PRICE_UNITS` and `PRICE_CURRENCY`?

Reply:
Now that I think of it, probably just dropping the `_ETH` suffix will work. Setting `PRICE` and `PRICE_SYMBOL` together should make it clear.